apps/src/code-studio/pd/workshop_enrollment/enroll_form.tsx

```diff
@@ -1,174 +1,199 @@
 
 interface SchoolInfoProps {
+  country?: string;
   school_id?: string;
-  school_district_name?: string;
-  school_district_other?: string;
   school_name?: string;
-  school_state?: string;
+  school_type?: string;
   school_zip?: string;
-  school_type?: keyof typeof SCHOOL_TYPES_MAPPING;
 }
 
 interface EnrollFormState {
-  attended_csf_intro_workshop?: keyof typeof ATTENDED_CSF_COURSES_OPTIONS;
-  csf_course_experience?: Partial<typeof CSF_COURSES>;
-  csf_courses_planned?: string[];
-  csf_intro_intent?: string;
-  csf_intro_other_factors?: string;
-  describe_role?: string;
-  email?: string;
-  confirm_email?: string;
-  explain_csf_course_other?: string;
-  explain_not_teaching?: string;
-  explain_teaching_other?: string;
-  first_name?: string;
-  grades_teaching?: string[];
-  last_name?: string;
-  planning_to_teach_ap?: string;
-  previous_courses: Array<string>;
-  role?: string;
-  taught_ap_before?: string;
-  years_teaching?: string;
-  years_teaching_cs?: string;
+  attended_csf_intro_workshop: keyof typeof ATTENDED_CSF_COURSES_OPTIONS;
+  confirm_email: string;
+  csf_course_experience: Partial<typeof CSF_COURSES>;
+  csf_courses_planned: string[];
+  csf_intro_intent: string;
+  csf_intro_other_factors: string[];
+  describe_role: string;
+  email: string;
+  explain_csf_course_other: string;
+  explain_not_teaching: string;
+  explain_teaching_other: string;
+  first_name: string;
+  grades_teaching: string[];
+  last_name: string;
+  planning_to_teach_ap: string;
+  previous_courses: string[];
+  role: string;
+  taught_ap_before: string;
+  years_teaching: string;
+  years_teaching_cs: string;
 }
 
-type CombinedFormState = keyof EnrollFormState | keyof SchoolInfoProps;
+type CombinedFormState = keyof EnrollFormState | 'school_info';
 
 type FormErrors = Partial<Record<CombinedFormState, string>>;
 
 type EnrollmentResponse = {
-  workshop_enrollment_status: string;
   account_exists: boolean;
-  sign_up_url: string;
   cancel_url: string;
+  sign_up_url: string;
+  workshop_enrollment_status: string;
 };
 
-type EnrollFormProps = EnrollFormState & {
+/**
+ * not all props are passed down from the parent WorkshopEnroll component
+ * but they are used in unit tests to set initial state
+ */
+type EnrollFormProps = {
   collect_demographics?: boolean;
+  email?: string;
+  first_name?: string;
+  last_name?: string;
+  role?: string;
   onSubmissionComplete: (response?: EnrollmentResponse) => void;
+  previous_courses: string[];
+  school_info?: SchoolInfoProps;
   user_id: number;
   workshop_course?: string;
   workshop_id: number;
   workshop_subject?: string;
-  school_info: SchoolInfoProps;
+  grades_teaching?: string[];
+  csf_intro_intent?: string;
+  attended_csf_intro_workshop?: string;
+  years_teaching?: string;
+  years_teaching_cs?: string;
+  taught_ap_before?: string;
+  planning_to_teach_ap?: string;
 };
 
+const Label = ({
```

Comment:
Sorry, should've asked for this this morning, but can you add a comment detailing why this is needed?